tracer-transformers/example2/Main.hs

```diff
@@ -50,8 +50,10 @@ main = do
 --- demo code
 
 -- a level of verbosity
-data Verbosity = Silent | Normal | Verbose
-                 deriving (Eq, Ord, Show)
+data Verbosity = Silent
+    | Normal
+    | Verbose
+    deriving (Eq, Ord, Show)
```

Comment:
This looks rather odd..

Reply:
:man_shrugging:  yes, right. can we turn off "stylish-haskell" for a region?

Reply:
it can't get better than that.